app-tasks/rf/src/rf/uploads/geotiff/factories.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, upload):
         self.visibility = Visibility.PRIVATE
         self.datasource = self._upload.datasource
         self.acquisitionDate = self._upload.metadata.get('acquisitionDate')
-        self.cloudCover = self._upload.metadata.get('cloudCover')
+        self.cloudCover = self._upload.metadata.get('cloudCover') or 0
```

Comment:
I think if you want to do a default here `self._upload.metadata.get('cloudCover', 0)` is the usual way in python